action.yml

```diff
@@ -10,6 +10,9 @@ inputs:
   use-32bit:
     description: 'Force using 32 bit version of cmake, instead of looking for 64 bit version'
     default: 'false'
+  arch-series:
+    description: 'Architecture series - could be x86, x86_64, arm, arm64 - defaults to arch of runner'
+    default: '${{ runner.arch }}'
```

Comment:
This is only here to force `runner.arch` to be available - if you know how to query it from ts please let me know and I'll rework it